src/main/archetype/ui.apps/src/main/content/jcr_root/apps/msm/__siteName___blueprint/.content.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file's path changes to a non-standard value if using `__siteName__` - meaning it could be `Customer Site`. This should use the `__appsFolderName__` so the resulting folder is `customer-site` instead.